lib/beaker/host/mac/pkg.rb

```diff
@@ -25,7 +27,35 @@ def generic_install_dmg(dmg_file, pkg_base, pkg_name)
     end
     dmg_name = File.basename(dmg_file, '.dmg')
     execute("hdiutil attach #{dmg_name}.dmg")
-    execute("installer -pkg /Volumes/#{pkg_base}/#{pkg_name} -target /")
+
+    # First check if the specific package exists, otherwise use wildcard
```

Comment:
IMHO this should at least be documented in the `@param` documentation for `pkg_name`. Perhaps it should even start to default to `nil` and this method should handle it.

Still, I wonder in which case you can't predict the `pkg_name` for a dmg.

Reply:
Hi @ekohl.  I updated the  `@param` documentation for `pkg_name` but didn't make it `nil` by default.  My thinking here is that `Mac::Pkg.install_package()` seems to be the "public" interface into this at which point it always passes the `pkg_name` to `generic_install_dmg()`.  You ok with this?